src/styles/components/nodes-table/styles.less

```diff
@@ -0,0 +1,38 @@
+& when (@nodes-tables-enabled) {
+  .nodes-table {
+    col.node-table--col-hostname {
```

Comment:
no need for such specificity `col.node-table--col-hostname` it can be written like so `node-table--col-hostname`

Same for the code below